pkg/fanal/types/const.go

```diff
@@ -57,6 +57,7 @@ const (
 	Pip            LangType = "pip"
 	Pipenv         LangType = "pipenv"
 	Poetry         LangType = "poetry"
+	Uv             LangType = "uv"
```

Comment:
Add this type to languages ​​and lock files.
Otherwise, this works for `rootfs`:
```
➜  trivy git:(feat/py-uv) ✗ ./trivy -q rootfs ./pkg/fanal/analyzer/language/python/uv/testdata/happy -f json --list-all-pkgs | jq '.Results[].Packages[].ID'
"uv-test@0.1.0"
"pluggy@1.5.0"
"requests@2.32.3"
"certifi@2024.8.30"
"charset-normalizer@3.4.0"
"colorama@0.4.6"
"idna@3.10"
"iniconfig@2.0.0"
"packaging@24.2"
"pytest@8.3.4"
"urllib3@2.2.3"

```

Reply:
Forgot about that. Done https://github.com/aquasecurity/trivy/pull/8080/commits/72a99a06cb5d320326cd90314515837531b5aabc